bot/tests/test_artifacts.py

```diff
@@ -102,12 +102,12 @@ def test_download(
     assert mocked_get_task_artifact.call_count == 1
     assert mocked_download_artifact.call_count == 2
     assert mocked_download_artifact.call_args_list[0] == mock.call(
-        "ccov-artifacts/linux_mochitest-devtools-chrome-4_code-coverage-grcov.zip",
+        "ccov-artifacts/linux_mochitest-4_code-coverage-grcov.zip",
```

Comment:
Why did this change?

Reply:
Here we use a [fixture describing a task ](https://github.com/mozilla/code-coverage/blob/master/bot/tests/fixtures/test_task_from_group.json#L126), with `mochitest` as `suite.name`.
So the generated local filename does not have the fully extended name anymore.

Reply:
As discussed on IRC, i reverted the unit tests by updating the fixture's suite
in a64e0a to `mochitest-devtools-chrome`